### PR TITLE
Bugfix/no task/providers medicare list

### PR DESCRIPTION
**Pull request by joako-gaona**
# ⚡ {0} - [{1}](https://app.clickup.com/t/{2}) ⚡

## 💻 What type of change is this?

{3}

## ⭐ Description

For more background, see ticket **{1}[in-progress].**

### Before

image

### After

image

### To review
- [ ] I have tested this change locally in multiple screen sizes
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] If my task include an endpoint, I add the endpoint to Hopscotch/Postman Project
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: resources/js/screens/ProvidersList.tsx [hunk 1]
```diff
@@ -17,30 +17,78 @@ const medicareDmeProviders: Record<
...
-      phoneNumber: "877-814-5459",
+      name: "Advanced Diabetes Supply",
+      email: "libreintakes@northcoastmed.com",
+      phoneNumber: "866-976-9110",
     },
     {
       id: 5,
       name: "Edgepark Medical Supplies",
-      email: "edgeparkcareteam@edgepark.com",
+      email: "diabetesreferrals@cardinalhealth.com",
```
Comment (sgoycoechea-lightit) [inline resources/js/screens/ProvidersList.tsx:47]: Is this email right for this name?

File: resources/js/screens/ProvidersList.tsx [hunk 2]
```diff
@@ -17,30 +17,78 @@ const medicareDmeProviders: Record<
...
+    {
+      id: 6,
+      name: "CCS Medical",
+      email: "libre@ccsmed.com",
+      phoneNumber: "800-951-1725",
+    },
+    {
+      id: 7,
+      name: "Byram Healthcare/Apria",
+      email: "Byram Healthcare/Apria",
```
Comment (sgoycoechea-lightit) [inline resources/js/screens/ProvidersList.tsx:59]: Is this email intended?

Comment (joako-gaona) [reply resources/js/screens/ProvidersList.tsx:59]: This one is wrong haha

File: resources/js/screens/ProvidersList.tsx (hunk 1 above)
Comment (joako-gaona) [reply resources/js/screens/ProvidersList.tsx:47]: yes

File: resources/js/screens/ProvidersList.tsx [hunk 3]
```diff
@@ -17,30 +17,78 @@ const medicareDmeProviders: Record<
...
+      id: 11,
+      name: "Bridgewater Health Supplies",
+      email: "info@bridgewaterhealthsupplies.com",
+      phoneNumber: "800-974-2055",
+    },
+    {
+      id: 12,
+      name: "Mini Pharmacy",
+      email: "info@minipharmacy.com",
+      phoneNumber: "info@minipharmacy.com",
```
[the remark this answers is not in the document]
Comment (joako-gaona) [reply resources/js/screens/ProvidersList.tsx:90]: omg sorryy

[the remark this answers is not in the document]
Comment (sgoycoechea-lightit) [reply resources/js/screens/ProvidersList.tsx:90]: don't be sorry 🤣

**Review by sgoycoechea-lightit — APPROVED** [review]

**Review by sgoycoechea-lightit — APPROVED** [review]

**Pull request merged**